vtparse/src/lib.rs

```diff
@@ -422,7 +422,7 @@ impl VTParser {
                 full: false,
             },
 
-            params: Default::default(),
+            params: [CsiParam::Integer(0); MAX_PARAMS],
```

Comment:
why is this change needed?

Reply:
Default::default() doesn't work on 256 element long arrays - at least, I couldn't get it to work.